pubsub/cloud-client/subscriber.py

```diff
@@ -284,7 +284,7 @@ def synchronous_pull(project_id, subscription_name):
     # Acknowledges the received messages so they will not be sent again.
     subscriber.acknowledge(subscription_path, ack_ids)
 
-    print("Received and acknowledged {} messages. Done.".format(NUM_MESSAGES))
+    print("Received and acknowledged {} messages. Done.".format(len(response.received_messages)))
```

Comment:
Two small things:

1. Line is over 80 column limit, causing lint to fail.
2. Although the previous string was using a doublequote, as per our style guide, prefer single-quote strings.